src/components/mapPageComponents/mapView/index.tsx

```diff
@@ -378,7 +378,7 @@ const MapView: React.FC<MapViewProps> = ({
           // Sets the information to display in the popup
           setActiveTreeInfo({
             id: siteId,
-            species: eventFeature.getProperty('species'),
+            species: eventFeature.getProperty('commonName'),
```

Comment:
Do you know why we use camelCase here and snake_case elsewhere (like `tree_present` and `block_id`)?

Reply:
There's no good reason for us to use snake case anywhere in the frontend